helm/odigos/templates/ui/role.yaml

```diff
@@ -28,6 +28,19 @@ rules:
       - update
       - delete
   {{- end }}
+  - apiGroups:
+      - apps
+    resources:
+      - replicasets
+    verbs:
+      - get
+      - list
+  - apiGroups:
+      - autoscaling
+    resources:
+      - horizontalpodautoscalers
+    verbs:
+      - get
```

Comment:
do we need to add these to the cli equivalent as well? or is it going down soon?

I guess if anyone is using the cli-based code manifests, this will create an error.